onnxruntime/core/providers/cuda/nn/dropout.cc

```diff
@@ -6,10 +6,23 @@
 namespace onnxruntime {
 namespace cuda {
 
+ONNX_OPERATOR_VERSIONED_KERNEL_EX(
+    Dropout,
+    kOnnxDomain,
+    12, 12,
```

Comment:
>  12, 12, [](start = 2, length = 9)

nit. can use ONNX_OPERATOR_KERNEL_EX?

Reply:
Seems for previous versions other than the latest, ONNX_OPERATOR_VERSIONED_KERNEL_EX is used, even it covers only one version. For example, Slice did the same thing for version 10.

Reply:
OK.